lib/Flags.js

```diff
@@ -0,0 +1,284 @@
+// @flow
+
+const Report = require('./Report.js');
+
+function help() /*: string */ {
+  return `
+elm-test init
+    Create example tests
+
+elm-test
+    Run tests in the tests/ folder
+
+elm-test TESTFILES
+    Run TESTFILES, for example "src/**/*Tests.elm"
+
+elm-test install PACKAGE
+    Like \`elm install PACKAGE\`, except it installs to
+    "test-dependencies" in your elm.json
+
+Options:
+    --compiler /path/to/compiler
+        Use a custom path to an Elm executable. Default: elm
+    --seed INT
+        Run with a previous fuzzer seed. Default: A random seed
+    --fuzz INT
+        Run with each fuzz test performing this many iterations. Default: 100
+    --report json
+    --report junit
+    --report console
+        Print results to stdout in the given format. Default: console
+    --version
+        Print version and exit
+    --watch
+        Run tests on file changes
+  `.trim();
+}
+
+const longOptionWithValue = /^(--[^-=][^=]*)=([^]*)$/;
+const looksLikeOption = /^--?[^-]/;
+
+// Poor man’s type alias. We can’t use /*:: type Command = ... */ because of:
+// https://github.com/prettier/prettier/issues/2597
+const Command /*:
+  | { tag: 'help' }
+  | { tag: 'version' }
+  | { tag: 'init' }
+  | { tag: 'install', packageName: string }
+  | { tag: 'make', testFileGlobs: Array<string> }
+  | { tag: 'test', testFileGlobs: Array<string> } */ = { tag: 'help' };
+void Command;
+
+function parseArgv(
+  argv /*: Array<string> */
+) /*: Result<
+  string,
+  {
+    command: typeof Command,
+    options: {
+      compiler: string | void,
+      fuzz: number,
+      report: typeof Report.Report,
+      seed: number,
+      watch: boolean,
+    },
+  }
+> */ {
+  const options = {
+    help: false,
+    version: false,
+    watch: false,
+    compiler: undefined,
+    seed: Math.floor(Math.random() * 407199254740991) + 1000,
```

Comment:
What is the source of this magic constant?

Reply:
I have no idea, but I can try to dig in the git history

Reply:
This is the first reference I can find: https://github.com/rtfeldman/node-test-runner/commit/276b05848ff0e210165acad1ce20943fc2316948#diff-6d021177521eff4c6fd312f2415f9a63604be87036870434271b5ec87e0bd2e1R633

I’m none the wiser, though

Reply:
probably copied from the `defaultSeed` function in elm/random, or something. We can leave this a mystery